components/compositing/webview_renderer.rs

```diff
@@ -395,6 +395,7 @@ impl WebViewRenderer {
         self.dispatch_input_event(event);
     }
 
+    /// Send touch event to the pipeline. Returns true if the event was send
```

Comment:
```suggestion
    /// Send a [`TouchEvent`] to the Constellation for this [`WebViewRenderer`]. 
    /// Returns true if the event was sent and false otherwise.
```